src/ui/wxWidgets/AddEditPropSheetDlg.h

```diff
@@ -388,10 +392,12 @@ class AddEditPropSheetDlg : public wxPropertySheetDialog
   wxString m_RMTime; // Any field modification time
   wxString m_AccessTime;
   wxString m_CreationTime;
-  wxString m_CurrentExpirationTime;
+  wxString m_OriginalExpirationDate;
   wxString m_ModificationTime;
+  wxRadioButton *m_OriginalButton;
   bool m_Recurring;
-  wxString m_ExpirationTime;
+  bool m_OriginalRecurring;
+  time_t m_OriginalDayttt;
```

Comment:
Not sure what the third "t" stands for, but for consistency with the other time_t member:
```suggestion
  time_t m_tttOriginalDay;
```

Reply:
Well...I'm going for consistency with the other m_Original's :)